plugins/auth-core/src/static/auth/abac.util.ts

```diff
@@ -9,6 +9,10 @@ export function getInvalidAttributes(
   // eslint-disable-next-line @typescript-eslint/ban-types
   data: Object
 ): string[] {
-  const filteredData = permission.filter(data);
+  // Without this, graphql queries will be denied access
+  // for anything requiring permissions 
+  // regardless of whether or not they have the required permissions.
+  const d = JSON.parse(JSON.stringify(data));
```

Comment:
@d-sonuga  Please change to :   const filteredData = permission.filter(JSON.parse(JSON.stringify(data))); 
instead of creating the new const d